xclim/indicators/land/_streamflow.py

```diff
@@ -5,13 +5,19 @@
 from xclim.core.cfchecks import check_valid
 from xclim.core.indicator import ResamplingIndicator
 from xclim.core.units import declare_units
-from xclim.indices import base_flow_index, generic, rb_flashiness_index
+from xclim.indices import (
+    base_flow_index,
+    generic,
+    rb_flashiness_index,
+    standardized_streamflow_index,
+)
 
 __all__ = [
     "base_flow_index",
     "doy_qmax",
     "doy_qmin",
     "rb_flashiness_index",
+    "standardized_streamflow_index",
```

Comment:
I'm surprised that there isn't a namespace collision here due to the shadowing of `standardized_streamflow_index`. My tendency is to import the indice with an underline, i.e.
```
from xclim.indices import standardized_streamflow_index as _standardized_streamflow_index
```
Then expose the Indicator without the underline. If this isn't emitting any warnings though, no worries.